### PR TITLE
vendor: update libslirp

### PR DESCRIPTION
**Pull request by giuseppe**
update to libslirp d171af3732a0610a25334b06b77fa547bd677918

I need it for https://gitlab.freedesktop.org/slirp/libslirp/merge_requests/19

Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

File: vendor.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -eux -o pipefail
 # Aug 26, 2019
```
Comment (AkihiroSuda) [inline vendor.sh]: please update date, then LGTM

**Comment by AkihiroSuda** [comment]
thanks, do you want to backport this to the v0.3 branch?

File: vendor.sh (hunk 1 above)
Comment (giuseppe) [reply vendor.sh]: fixed

**Comment by giuseppe** [comment]
> thanks, do you want to backport this to the v0.3 branch?

we can just cut a new minor release, and in Fedora/RHEL we'll use it

**Comment by giuseppe** [comment]
thanks for the quick review!  Could we cut a new release?

**Comment by AkihiroSuda** [comment]
Let's cut a new release after libslirp v4.1.0 gets released: https://gitlab.freedesktop.org/slirp/libslirp/merge_requests/21

**Pull request merged**